data/src/main/java/org/oppia/android/data/backends/gae/NetworkInterceptor.kt

```diff
@@ -22,10 +22,10 @@ class NetworkInterceptor @Inject constructor() : Interceptor {
     val response = chain.proceed(request)
 
     if (response.code() == Constants.HTTP_OK) {
-      if (response.body() != null) {
-        var rawJson = response.body()!!.string()
+      response.body()?.let {
+        var rawJson = it.string()
```

Comment:
Instead of using `it` could you something more descriptive maybe
`resBody` or simply  `body`